### DISCUSSION TITLE
Project Display

### DISCUSSION THREAD

**Discussion by sureshkumar2704** [community/community#133837] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Can anyone suggest how to display the project in profile page ?

**Comment by kavin-raja-d** [comment]
Select the repository and click three dot and then mark as pin.

**Comment by ebndev** [comment, marked as answer]
Hi @sureshkumar2704 & @kavinraja-developer, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
